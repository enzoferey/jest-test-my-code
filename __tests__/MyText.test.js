import React from "react";
import renderer from "react-test-renderer";

const MyText = global.MyText;

test("snapshot matches", () => {
  const tree = renderer.create(<MyText name="Enzo" />).toJSON();
  expect(tree).toMatchSnapshot();
});
