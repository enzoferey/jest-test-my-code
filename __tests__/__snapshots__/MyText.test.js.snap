// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot matches 1`] = `
<p>
  Hello $
  Enzo
</p>
`;
